modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
+
+Autoscaling is dependent on the resource quotas that are set. If there are insufficient resources, worker nodes are removed from cluster autoscaling when pods fail. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
+
+[NOTE]
+====
+Autoscaling can be disabled on a machine pool, but it cannot be disabled on a cluster. However, if no machine pool autoscaling is enabled, then autoscaling will not occur on that cluster even if autoscaling is enabled on the cluster.
+====
+
+.Procedure
+
+. To identify the machine pool IDs in a cluster, enter the following command:
```

Comment:
seems like machine pools should be capitalized like in OCM? "Machine pool".

Reply:
Per OpenShift doc guidelines [1], general references to objects should be lowercase, so "machine pool" is correct.

[1] https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-general-references

Reply:
thanks for sharing.